tests/tenant.test.ts

```diff
@@ -0,0 +1,15 @@
+import {test, expect} from '@playwright/test';
+
+test.describe('Test Tenants page', async () => {
+    test('Tenants page is OK', async ({page}) => {
+        const response = await page.goto('/tenants');
```

Comment:
good for start, later pls take a look at Page Object Model approach - afaik it's recommended by playwright maintainers